src/App.tsx

```diff
@@ -3,12 +3,31 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  moviesList: Movie[],
+  query: string;
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    moviesList: moviesFromServer,
+    query: '',
+  };
+
+  filteredList = (movie: Movie) => {
```

Comment:
not meaningful name of the method. Better to call it "shouldBeFiltered" or something like that.